src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -145,3 +159,16 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
         return !!version && gte(version, '11.0.0');
     }
 }
+
+async function validateDatabaseName(name: string, getChildrenTask: Promise<AzExtTreeItem[]>): Promise<string | undefined | null> {
+    if (!name || name.length < 1) {
```

Comment:
nit: `!name` already handles the empty string case. You don't have to check the string length